components/supervisor/frontend/src/index.ts

```diff
@@ -142,19 +141,6 @@ const toStop = new DisposableCollection();
                     return document.body;
                 }
             }
-            if (instance.status.phase === 'stopped') {
-                stopped = true;
-            }
-            if (stopped && (
-                instance.status.phase === 'preparing' ||
-                instance.status.phase === 'pending' ||
-                instance.status.phase === 'creating' ||
-                instance.status.phase === 'initializing')) {
-                // reload the page if the workspace was restarted to ensure:
```

Comment:
@geropl, how relevant is this removal?

Reply:
@AlexTugarev see discussions here: https://github.com/gitpod-io/gitpod/pull/8125#discussion_r804569170

I think it is the proper way to go.

Reply:
We do it on purpose to remove dependencies from supervisor and loading screen. Now a user will need to reload explicitly. I think it is fine.

Reply:
Thanks, I missed that, as it was resolved. 

Nice and clean!